src/docfx/template/MustacheTemplate.cs

```diff
@@ -72,7 +71,13 @@ private class PartialLoader : IStubbleLoader
 
             public string Load(string name) => _cache.GetOrAdd(
                 name,
-                key => new Lazy<string>(() => File.ReadAllText(Path.Combine(_dir, name + ".tmpl.partial")))).Value;
+                key => new Lazy<string>(() =>
+                {
+                    var fileName = Path.Combine(_dir, name);
+                    if (File.Exists(fileName))
+                        return File.ReadAllText(fileName);
+                    return File.ReadAllText(Path.Combine(_dir, name + ".tmpl.partial"));
```

Comment:
NetMember has full-name partial path defination
```
<td>
  <xref uid="{{ uid }}" template="partials/dotnet/xref-name.tmpl" />
</td>
```

Reply:
Do you prefer to change `docs-ui` or here?

Reply:
I'd like to, this will make docs-ui partial style more consistent.
But this also requires v2  to support it, which may need some tests first.

Reply:
So then we'll make then change later as we onboard .NET?

Reply:
@yufeih Talked with @superyyrrzz , This will break v2 xref process.
Also perform a test without '.tmpl' ext and rename the .tmpl file.
[Report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2020%5C3%5C24%5Cdde987a0-8603-7cdc-14a2-f3823991e8d9%5CCommit%5C202003240833006394-template-xref-partial%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZTCH0QPBZt16HsxdsPjYWkugF0HN3TkpjaZdfyUTfdY%3D&st=2020-03-24T08%3A30%3A41Z&se=2020-04-24T08%3A35%3A41Z&sp=r)

[template](https://github.com/microsoft/templates.docs.msft/commits/mingwli/xref-partial)
[test-repo](https://github.com/mingwli/test/blob/template-xref-partial/.openpublishing.publish.config.json#L69)
```diff
<tr>
    <td>
-        <a data-linktype="relative-path" href="catlibrary.cat-2.catchjerry#catlibrary_cat_2_catchjerry">catchjerry()</a>
+        <a class="xref" href="catlibrary.cat-2.catchjerry#CatLibrary_Cat_2_CatchJerry" data-linktype="relative-path">CatchJerry()</a>
    </td>
    <td>
        <p>this one can't.</p>
    </td>
</tr>
```